benchmarks/scripts/final.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This file is identical to clean.sh

Reply:
Sure but this shell script is needed to run the slurm job.